app/installer.sh

```diff
@@ -159,3 +168,42 @@ else
   printf "ERROR\tUnable to download %s\n" "$TARGET_NAME"
   exit 1
 fi
+
+# Sets `VERSION` to the git tag representing the latest preview version.
+set_preview_version() {
+  if [[ ! -x `which gh` ]]; then
+    echo "You must have github's gh client installed to install a preview version"
+    exit 1
+  fi
+
+  JQCMD='.data.repository.ref.target.history.nodes | map(select(.statusCheckRollup.state == "SUCCESS")) | first | .oid'
+  PREVIEW_COMMIT=`gh api graphql  -f owner=${owner} -f repo=${repo} -f branch=${BRANCH} --jq "${JQCMD}" -F query='
+  query($owner: String!, $repo: String!, $branch: String!) {
+    repository(owner: $owner, name: $repo) {
+      ref(qualifiedName: $branch) {
+        target {
+          ... on Commit {
+          history(first: 10) {
+            nodes {
+              oid
+              statusCheckRollup {
+                state
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  '`
+
+  echo "Getting release $VERSION"
+
+  if [ "$REPO_PREFIX" == "kong-mesh" ]; then
+    PREVIEW_COMMIT=${PREVIEW_COMMIT:0:7}
+  else
+    PREVIEW_COMMIT=${PREVIEW_COMMIT:0:9}
+  fi
```

Comment:
I did it this way because the short commit hash isn't constant (we should use `git rev-parse --short=9` (we could potentially fix the build instead :) ).

Let's fix this in Kuma and get rid of the if block

Reply:
For now I replaced this with a sh-compatible version, but that would be nice to sort out in Kuma/Kong Mesh.